docs/samples/drift-detection/alibi-detect/cifar10/cifar10_drift.ipynb

```diff
@@ -13,8 +13,9 @@
     "\n",
```

Comment:
<p>seems getting an error here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/kfserving/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>How are you testing the next cell shows how you can use in the case you don't have a load balancer.</p>